client/components/UI/TermsList.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {gettext} from 'superdesk-core/scripts/core/utils';
+import {gettext} from 'superdesk-core/scripts/core/utils.tsx';
```

Comment:
This import can be replaced with the SuperdeskAPI, and use `localization.gettext` in the render function

Reply:
We won't need this PR since Petr decided that we use post-broadcasting branch instead. So I'm closing this PR.